src/key.cpp

```diff
@@ -333,6 +333,7 @@ bool CKey::Derive(CKey& keyChild, ChainCode &ccChild, unsigned int nChild, const
 }
 
 bool CExtKey::Derive(CExtKey &out, unsigned int _nChild) const {
```

Comment:
Maybe mark as `[[nodiscard]]` in the header?

Reply:
Done, and fixed two missing occurrences (in the descriptor tests) this revealed. Thanks!